dev-python/dnspython/dnspython-1.16.0-r2.ebuild

```diff
@@ -23,6 +23,11 @@ RDEPEND="dev-python/pycryptodome[${PYTHON_USEDEP}]
 	!dev-python/dnspython:py2
 	!dev-python/dnspython:py3"
 
+PATCHES=(
+	# Mutable mapping was moved to a different module in 3.10 and removed from the old one
+	"${FILESDIR}"/abc-mutable-mapping.patch
```

Comment:
Start the patch name with `${P}-`, I'd go for simple `${P}-py310.patch`.

Reply:
Done